test/unit/module-register/register.mjs

```diff
@@ -0,0 +1,5 @@
+import { register } from 'module';
+import { pathToFileURL } from 'url';
+
+register('./hook.mjs', import.meta.url);
+register('./hook2.mjs', pathToFileURL(__filename));
```

Comment:
How about a test for `pathToFileURL('./')`?

Reply:
I also want to add a test for the bare specifier code path too